index.js

```diff
@@ -53,36 +77,48 @@
 
 
 // ***** Question 5 *****
+function updateGrade(student, grade){
+  student.grade = grade
+}
+
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 5", "color: red")
-// const student1 = { name: "Duane", grade: 88 }
-// updateGrade(student1, 92)
-// console.log(student1)
+ console.log("%cQuestion 5", "color: red")
+ const student1 = { name: "Duane", grade: 88 }
+ updateGrade(student1, 92)
+ console.log(student1)
 // // => { name: "Duane", grade: 92 }
-// console.log("%c----------", "color: red")
+ console.log("%c----------", "color: red")
 
 
 
-// ***** Question 6 *****
+// ***** Question 6 *****{
+function printNameAndPhones(users){
+  users.forEach(function (user){
+    console.log(`${user.name}`)
+    console.log(`Cell: ${user.phones.cell}`)
+    console.log(`Office: ${user.phones.office}`)
+      }
+    )
+  }
```

Comment:
Looks solid. Nice job transforming things we were comfortable with in Ruby to their JS equivalent.

- .each  (vs) .forEeach
- bracket notation (vs) object dot notation
- "#{}" (vs) `${}`